aurget

```diff
@@ -473,23 +477,26 @@ print_info() {
 
     ((outofdate)) && ood="${colorR}Yes$nocolor"
 
-    printf "${colorW}Repository  : ${colorM}aur$nocolor\n"
-    printf "${colorW}Name        : $name$nocolor\n"
+    printf "%bRepository  : %baur%b\n" "${colorW}" "${colorM}" "${nocolor}"
+    printf "%bName        : %s%b\n" "${colorW}" "${name}" "${nocolor}"
     [[ "$name" != "$pkgbase" ]] &&
-      printf "${colorW}Base        : $colorB$pkgbase$nocolor\n"
-    printf "${colorW}Version     : $colorG$version$nocolor\n"
-    printf "${colorW}URL         : $colorC$url$nocolor\n"
-    printf "${colorW}Out of date : $nocolor$ood\n"
-    printf "${colorW}Votes       : $nocolor$votes\n"
-    printf "${colorW}Description : $nocolor$description\n\n"
+      printf "%bBase        : %b%s%b\n" "${colorW}" "${colorB}" "${pkgbase}" "${nocolor}"
+    printf "%bVersion     : %b%s%b\n" "${colorW}" "${colorG}" "${version}" "${nocolor}"
+    printf "%bURL         : %b%s%b\n" "${colorW}" "${colorC}" "${url}" "${nocolor}"
+    printf "%bOut of date : %b%b%b\n" "${colorW}" "${nocolor}" "${ood}" "${nocolor}"
+    printf "%bVotes       : %b$votes\n" "${colorW}" "${nocolor}"
+    printf "%bDescription : %b%s\n\n" "${colorW}" "${nocolor}" "${description}"
   done
 }
 # }}}
 
 # Add available upgrades {{{
 add_available_upgrades() {
+  local update_packages
   info 'Starting AUR upgrade...'
-  arguments=($(available_upgrades) "${arguments[@]}")
+
+  mapfile -t update_packages < <(available_upgrades)
+  arguments+=("${update_packages[@]}")
```

Comment:
This actually changed behavior. Previously the result was `arguments = upgrades + arguments` but now it's `arguments = arguments + upgrades`. I don't know if I care, but I think I was trying to match what `pacman -Syu x y z` does in terms of processing available upgrades then command line arguments :man_shrugging:

Reply:
That is something, that should probably be changed again, I will think of a solution :-) 
It is probably the better thing to do, first upgrade the installed packages then install new ones.